ql/ql/src/codeql_ql/ast/internal/Predicate.qll

```diff
@@ -162,17 +176,20 @@ module PredConsistency {
     c > 1 and
     resolvePredicateExpr(pe, p)
   }
+  // This can happen with parametarized modules
```

Comment:
```suggestion
  // This can happen with parameterized modules
```

I wonder, could we resurrect this test (and others) if we filter out stuff related to parameterised modules?

Reply:
We could.... 
We can check if one of the targets is a `signature predicate`, and then allow multiple if that's the case.